src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.ts

```diff
@@ -77,10 +79,25 @@ export class CustomerListComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   editCustomer(customerId: string) {
+    this.idToEdit = customerId;
+    if (this.haveChanges === true) {
```

Comment:
Does the comparison need to be explicit?
`this.haveChanges === true`

Reply:
I think it is not necessary I will check it thanks